framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1457,9 +1457,17 @@ public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree lhsTree) {
     /**
      * Returns the type of a varargs array of a method invocation or a constructor invocation.
      *
+     * <p>A null return value might mean:
+     *
+     * <ul>
+     *   <li>This type-checker is not using dataflow analysis.
+     *   <li>The given tree is dead code, so dataflow analysis never reached it to refine its type.
```

Comment:
This is incorrect.  This method does not use the refined type from dataflow. Rather it use the synthetic NewArrayTree created by dataflow and computes a type of that NewArrayTree via the AnnotatedTypeFactory.  It does not return null if the method call is in dead code.

This method only returns null if useFlow is false.